man/systemd.network.xml

```diff
@@ -3491,6 +3491,14 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MPUBytes=</varname></term>
+        <listitem>
+          <para>Rounds each packet (including overhead) up to the specified bytes. Takes an unsigned
+          integer in the range 0…256. Defaults to unset and kernel's default is used.</para>
```

Comment:
hmm, the patch actually implements things so that "0" means "leave kernel default", i.e. it has special meaning. Thus this should document 1…256 here I guess.